src/components/LoginForm/LoginForm.jsx

```diff
@@ -19,11 +19,11 @@ const LoginForm = ({ errorMessage, setErrorMessage }) => {
     return errors;
   };
 
-  const handleLoginSubmit = (values) => {
+  const handleLoginSubmit = (events) => {
```

Comment:
We get **a** event not events.
```suggestion
  const handleLoginSubmit = (event) => {
```